application/libraries/PluginManager/PluginBase.php

```diff
@@ -251,13 +251,12 @@ protected function registerSetting($name, $options = array('type' => 'string'))
     }
 
     /**
-     *
      * @param array<string, mixed> $settings
      * @return void
      */
-    public function saveSettings($settings)
+    public function saveSettings($aSettings)
```

Comment:
Hm why this change? Hungarian notation should be used less, not more :stuck_out_tongue:

Reply:
Because : it must be set here because it was set in LimeSurvey\PluginManager\iPlugin::saveSettings

See https://github.com/LimeSurvey/LimeSurvey/runs/4444187513?check_suite_focus=true#step:9:34

>  ERROR: ParamNameMismatch - application/core/plugins/ExpressionAnswerOptions/ExpressionAnswerOptions.php:93:34 - Argument 1 of ExpressionAnswerOptions::saveSettings has wrong name $settings, expecting $aSettings as defined by LimeSurvey\PluginManager\iPlugin::saveSettings (see https://psalm.dev/230)
>    public function saveSettings($settings)

Reply:
lol ok, thanks

Reply:
Rename LimeSurvey\PluginManager\iPlugin::saveSettings parameters seem better … i fix it